docs/key-issues/gpus.md

```diff
@@ -80,9 +80,9 @@ provides a number of packages:
 adds maintenance overhead to project developers and consumes more storage and
 network bandwidth for PyPI.org. Moreover, it also prevents downstream projects
 from properly declaring the dependency unless they also follow a similar
-multi-package approach. As of CUDA 11, CUDA promises [Minor Version
+multi-package approach. As of CUDA 11, CUDA promises [CUDA Enhanced
```

Comment:
Minor Version Compatibility is the name of the link and the section title, and seems clearer. CUDA Enhanced Compatibility doesn't seem mentioned at all there? Or is this no longer the best link to use?

Reply:
Yeah... there's a bunch of confusion across different documents with naming and what features mean what. Let me try and clean this up.